draft-ietf-quic-tls.md

```diff
@@ -237,12 +237,12 @@ Data is protected using a number of encryption levels:
 - Handshake Keys
 - Application Data (1-RTT) Keys
 
-Application Data may appear only in the Early Data and Application Data
-levels. Handshake and Alert messages may appear in any level.
+Application Data can only appear in the early data and Application Data
+levels. Handshake and alert messages may appear in any level.
```

Comment:
1. Why is `Application Data` capitalized, but not `Early Data`? I don't personally care either way, but it seems inconsistent.
2. On the `may appear` to `can only appear` change, I personally preferred the old text. A QUIC implementation could theoretically (and incorrectly) send a `STREAM` frame in an Initial or Handshake packet, but it's not allowed. So I like `may` because it indicates it's not allowed. `can` makes it sound like it's not even possible.

Just my two cents.

Reply:
I agree.  I've changed to use lowercase in a few more places (basically everywhere that is not a diagram).